specs/rigil/suave-chain.md

```diff
@@ -98,11 +98,11 @@ type SuaveTransaction struct {
 
 ## TransactionRequest Serialization & Signing
 
-Transactions sent by users of SUAVE can take on two forms: 
+Messages sent by users of SUAVE can take on two forms: 
 1. Standard (legacy) Ethereum transaction
 2. `ConfidentialComputeRequest`
 
-Standard transactions are used to transfer SUAVE-ETH and deploy smart contracts to SUAVE. ConfidentialComputeRequests are a new [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) transaction type, used to interact with SUAVE smart contracts.
+Standard transactions are used to transfer SUAVE-ETH and deploy smart contracts to SUAVE chain. ConfidentialComputeRequests are a new [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) message type, used to interact with SUAVE smart contracts.
```

Comment:
Right, apologies for lack of explanation.

The name "transaction" comes from databases and thus is associated with a change in state. The blockchain community have also come to view transaction as "a message to interact with the blockchain."

Since CCR's don't change state and we explicitly want to distinguish messages to kettles from messages to suave chain, it is important to linguistically distinguish between transactions and non-transactions.

Reply:
sweet yes totally agree! was asking if you could add some of that explanation to the text in the spec if possible!

Reply:
how about now?